### DISCUSSION TITLE
AMP: Halaman Sub-Page yang Dijadikan Sebagai Halaman Utama Tidak Bisa di Direct ke Halaman AMP

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#195] [Q&A]
kang, halaman yang sub-page yang dijadikan sebagai halaman utama gak bisa di direct ke halaman AMP?

`https://namadomain.com` (domain utama)

`https://namadomain.com/article` (sub-page) => dijadikan sebagai halaman utama, pas di direct ke halaman AMP tidak bisa kang, di Google Search Console langsung muncul error `URL AMP mengarah ke halaman non-AMP`

`https://namadomain.com/amp/article` ke-direct ke `https://namadomain.com`

**Comment by taufik-nurrohman** [comment]
Yang nge-direct siapa kalau begitu? Apa di laman sub tersebut masih ada tag meta `rel="amphtml"`?
